.test_database_urls/postgres_14

```diff
@@ -0,0 +1,2 @@
+export TEST_DATABASE_URL="postgresql://postgres:prisma@localhost:5437"
+set -e TEST_SHADOW_DATABASE_URL
```

Comment:
The `set -e` crashes zsh (@do4gr  experienced it as well). `unset` seem to be posix standard, are you sure it doesn't work in fish? We should find a solution.